pystruct/learners/n_slack_ssvm.py

```diff
@@ -12,13 +12,17 @@
 import cvxopt
 import cvxopt.solvers
 
-from sklearn.externals.joblib import Parallel, delayed
+from sklearn.externals.joblib import Parallel, delayed, cpu_count
 from sklearn.utils import gen_even_slices
 
 from .ssvm import BaseSSVM
 from ..utils import unwrap_pairwise, find_constraint
 
 
+def find_constraint_map(args):
+    return find_constraint(* args)
```

Comment:
I'm a bit weary of messing with ``set_state`` and ``get_state`` but I'm not sure there is a way around it.
Otherwise that sounds good.

Why do you want to initialize the pool at the beginning of ``fit`` instead when the parallelization is actually used?